documentation/jetty-documentation/src/main/asciidoc/operations-guide/modules/module-ssl-reload.adoc

```diff
@@ -22,3 +22,5 @@ The module properties are:
 ----
 include::{JETTY_HOME}/modules/ssl-reload.mod[tags=documentation]
 ----
+
+The `resolveAlias` property is used to specify whether aliases should be resolved in the path of the KeyStore. If set to false and the path of the KeyStore is a symbolic link, the scanner will monitor the symbolic link file for changes instead of its target.
```

Comment:
I would rename the property to `followLinks` because that's how it is named in the JDK (and used by `Scanner`), it's easier to understand, and it is not related to our own internal concept of "alias".

Also, every sentence on a new line.